src/functionalTest/java/uk/gov/hmcts/dts/fact/admin/AdminCourtAreaOfLawEndpointTest.java

```diff
@@ -0,0 +1,158 @@
+package uk.gov.hmcts.dts.fact.admin;
+
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.dts.fact.model.admin.AreaOfLaw;
+import uk.gov.hmcts.dts.fact.util.AdminFunctionalTestBase;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static org.springframework.http.HttpStatus.*;
+import static uk.gov.hmcts.dts.fact.util.TestUtil.BEARER;
+import static uk.gov.hmcts.dts.fact.util.TestUtil.objectMapper;
+
+@ExtendWith(SpringExtension.class)
+public class AdminCourtAreaOfLawEndpointTest extends AdminFunctionalTestBase {
+
+    private static final String ADMIN_COURTS_ENDPOINT = "/admin/courts/";
+    private static final String AREAS_OF_LAW_PATH = "courtAreasOfLaw";
+    private static final String AYLESBURY_COUNTY_COURT_AND_FAMILY_COURT_SLUG = "aylesbury-county-court-and-family-court/";
```

Comment:
@ashwini-mv The string constant is named a court slug but you are adding a "/" at the end. You can remove the "/" here and add that to the beginning of AREAS_OF_LAW_PATH